src/applications/vaos/tests/services/appointment/transformers.unit.spec.js

```diff
@@ -616,7 +616,7 @@ describe('VAOS Appointment transformer', () => {
         });
 
         describe('va video appointment', () => {
-          describe('should return false after if video and less than 240 min ago', () => {
+          it('should return false after if video and less than 240 min ago', () => {
```

Comment:
These were running, but because they weren't `it`s, when they failed they threw the error in a weird way.